src/Main.cpp

```diff
@@ -17,7 +17,7 @@ int main(int argc, char const* argv[])
     }
 
     dvl_teledyne::Driver driver;
-    driver.open(argv[1]);
+    driver.openURI(argv[1]);
```

Comment:
You would need to start the acquisition now manually right?
Can you document the intended usage of this driver (open, configuration, start acquisition) in the header of the driver?

Reply:
Done.

Reply:
You still would need to start the acquisition here in order to read samples right?
Same for the dvl_teledyne_info executable. And lets also stop the acquisition again in all executables.

Reply:
Also done.